CHANGES.rst

```diff
@@ -10,6 +10,188 @@
 
 .. towncrier release notes start
 
+3.9.2 (2024-01-28)
+==================
+
+Bug fixes
+---------
+
+- Fixed websocket connection leak.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7978`.
+
+
+
+- Fixed `web.FileResponse` doing blocking I/O in the event loop.
```

Comment:
Client or server?

Also, it's linkable:

```suggestion
- Fixed :py:class:`web.FileResponse <aiohttp.web.FileResponse>` doing blocking I/O in the event loop.
```

Reply:
There's no reference documentation for this currently. So, can't link it.

Reply:
I suppose it wasn't backported, because the `latest` RTD build does have this reference.